src/hooks/useLogDownloader/index.ts

```diff
@@ -142,14 +144,19 @@ const useLogDownloader = (
           });
           setIsLoading(false);
         });
+    } else if (!url && !isLoadingEvergreen) {
+      dispatchToast.error("Log URL not specified, unable to download.", true, {
+        shouldTimeout: false,
+      });
+      setIsLoading(false);
```

Comment:
Possibly a silly question but would you mind describing what this part does? When I tried removing it, I could still load the logs 😮

Reply:
You can see the failed run [here](https://spruce.mongodb.com/task/parsley_ubuntu2204_large_e2e_test_patch_d9645fc8c24e8bed33c4309c376eb80360adc3b0_654bbfaa61837db75b414fa0_23_11_08_17_04_43?execution=0&sortBy=STATUS&sortDir=ASC); basically if there's no URL to download, we never enter the `if (url)` block, and Parsley is stuck in an infinite "downloading" state.

It would probably make more sense to only show that state on Parsley once the download actually begins/a URL is present, but as-is this was a more straightforward fix.